### PR TITLE
Add `osu!(lazer) Updates: November 15, 2024` news post

### PR DESCRIPTION
**Pull request by Walavouchey**
[web preview](https://osu1.roanh.dev/home/news/2024-11-15-osulazer-updates-november-15-2024)

### CODE REVIEW

File: news/2024/2024-11-15-osulazer-updates-november-15-2024.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+
+Below you will find [peppy](https://osu.ppy.sh/users/2) and [pishifat](https://osu.ppy.sh/users/3178418) discussing the rollout as well as all the new neat changes in the game.
+
+<div align="center">
+    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://www.youtube.com/embed/YKljRdxfhxQ" frameborder="0" allowfullscreen></iframe>
+</div>
+
+## Editor changes
+
+- **Added keyboard shortcut for cycling grid types (`Shift` + `G`).** Did you know we have not just square, but triangular and circular grids as well?
```
Comment (0x84f) [inline news/2024/2024-11-15-osulazer-updates-november-15-2024.md:23]:
```suggestion
- **Added a keyboard shortcut for cycling grid types (`Shift` + `G`).** Did you know we have not just square, but triangular and circular grids as well?
```

File: news/2024/2024-11-15-osulazer-updates-november-15-2024.md [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+- **Toggles and shortcuts for hitsound sample addition banks.** Seasoned mappers will likely be familiar with the `Shift`/`Alt` + `Q`/`W`/`E`/`R` hitsounding keys already, which now all work on osu!(lazer).
```
Comment (0x84f) [inline news/2024/2024-11-15-osulazer-updates-november-15-2024.md:24]:
```suggestion
- **Toggles and shortcuts for hitsound sample addition banks have been added.** Seasoned mappers will likely be familiar with the `Shift`/`Alt` + `Q`/`W`/`E`/`R` hitsounding keys already, which now all work on osu!(lazer).
```

File: news/2024/2024-11-15-osulazer-updates-november-15-2024.md [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+
+<div align="center" class="osu-md__paragraph">
+    <video width="95%" controls>
+        <source src="https://assets.ppy.sh/media/news/2024-11-16-relative-skin-size.mp4" type="video/mp4" preload="none">
+    </video>
+</div>
+
+## Other changes
+
+- **New osu!mania ruleset selection sample.** Our audio engineer [nekodex](https://osu.ppy.sh/users/102) has been at it again. Take a listen:
```
Comment (0x84f) [inline news/2024/2024-11-15-osulazer-updates-november-15-2024.md:50]:
```suggestion
- **The osu!mania ruleset selection sample has been reworked.** Our audio engineer [nekodex](https://osu.ppy.sh/users/102) has been at it again. Take a listen:
```

File: news/2024/2024-11-15-osulazer-updates-november-15-2024.md [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+date: 2024-11-16 23:00:00 +0000
+---
+
+This update contains a lot of quality-of-life changes. Feel free to continue your day.
+
+![](/wiki/shared/news/2024-11-16-osulazer-updates-november-15-2024/banner.jpg)
+
+For those who want to pause for a moment to read, here's the complete rundown:
+
+After the [last update](https://osu.ppy.sh/home/news/2024-10-09-osulazer-updates-october-9), an [update to performance algorithms](https://osu.ppy.sh/home/news/2024-10-28-performance-points-star-rating-updates) was rolled out to all players. If your global rank went up, congratulations. If not, maybe it's time to get back on the grind.
```
Comment (Hiviexd) [inline news/2024/2024-11-15-osulazer-updates-november-15-2024.md:13]:
```suggestion
After the [last update](https://osu.ppy.sh/home/news/2024-10-09-osulazer-updates-october-9), an [update to performance algorithms](https://osu.ppy.sh/home/news/2024-10-28-performance-points-star-rating-updates) was rolled out to all players. If your global rank went up, congratulations! If not, maybe it's time to get back on the grind.
```

File: news/2024/2024-11-15-osulazer-updates-november-15-2024.md [hunk 5]
```diff
@@ -0,0 +1,94 @@
...
+After:
+
+<div class="osu-md__paragraph">
+    <audio controls>
+        <source src="https://assets.ppy.sh/media/news/2024-11-16-ruleset-select-mania-after.mp3">
+    </audio>
+</div>
+
+- **You can now search chat channels and the friends/online user listing.** Popular and social people will appreciate this one.
+- **New `ranked` and `submitted` date filtering in song select.** This already existed on the website. See the [beatmap search article](/wiki/Beatmap_search) for an extensive list of filters.
```
Comment (0x84f) [inline news/2024/2024-11-15-osulazer-updates-november-15-2024.md:69]:
```suggestion
- **New `ranked` and `submitted` date filters can now be used in song select.** This already existed on the website. See the [beatmap search article](/wiki/Beatmap_search) for an extensive list of filters.
```

File: news/2024/2024-11-15-osulazer-updates-november-15-2024.md [hunk 6]
```diff
@@ -0,0 +1,94 @@
...
+- **Searching mods by acronym will highlight the corresponding mod for selection.** `F1` + mod acronym + `Enter` + `F1`. Now if only I could remember all the new mod acronyms...
+
+<div align="center" class="osu-md__paragraph">
+    <video width="95%" controls>
+        <source src="https://assets.ppy.sh/media/news/2024-11-16-mod-acronym-search.mp4" type="video/mp4" preload="none">
+    </video>
+</div>
+
+- **New settings button to reset all beatmap offsets.** Useful when you realise you've set all your offsets wrong.
```
Comment (0x84f) [inline news/2024/2024-11-15-osulazer-updates-november-15-2024.md:78]:
```suggestion
- **A new settings button has been added to reset all beatmap offsets.** Useful when you realise you've set all your offsets wrong.
```

File: news/2024/2024-11-15-osulazer-updates-november-15-2024.md [hunk 7]
```diff
@@ -0,0 +1,94 @@
...
+Below you will find [peppy](https://osu.ppy.sh/users/2) and [pishifat](https://osu.ppy.sh/users/3178418) discussing the rollout as well as all the new neat changes in the game. Below that, you'll find the most notable changes in text form.
+
+<div align="center">
+    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://www.youtube.com/embed/YKljRdxfhxQ" frameborder="0" allowfullscreen></iframe>
+</div>
+
+## Editor changes
+
+- **Added keyboard shortcut for cycling grid types (`Shift` + `G`).** Did you know we have not just square, but triangular and circular grids as well?
+- **Toggles and shortcuts for hitsound sample addition banks.** Seasoned mappers will likely be familiar with the `Shift`/`Alt` + `Q`/`W`/`E`/`R` hitsounding keys already, which now all work on osu!(lazer).
```
Comment (Hiviexd) [inline news/2024/2024-11-15-osulazer-updates-november-15-2024.md:24]: would be nice to showcase the new UI for this in a screenshot

File: news/2024/2024-11-15-osulazer-updates-november-15-2024.md [hunk 8]
```diff
@@ -0,0 +1,94 @@
...
+
+<div align="center" class="osu-md__paragraph">
+    <video width="95%" controls>
+        <source src="https://assets.ppy.sh/media/news/2024-11-16-mod-acronym-search.mp4" type="video/mp4" preload="none">
+    </video>
+</div>
+
+- **New settings button to reset all beatmap offsets.** Useful when you realise you've set all your offsets wrong.
+- **Local scores are now preserved even when deleting maps.** Quality-of-life change for maintainers of large beatmap libraries.
+- **New context menu option for copying link to an online score.** The same thing exists when right-clicking beatmaps in song select, btw.
```
Comment (0x84f) [inline news/2024/2024-11-15-osulazer-updates-november-15-2024.md:80]:
```suggestion
- **Copying links to plays can now be done by right-clicking on them from the song select screen.** The same option exists for beatmaps in song select, btw.
```

**Review by 0x84f — COMMENTED** [review]
i feel this gives them a bit more personality rather than just declaring what the change is

File: news/2024/2024-11-15-osulazer-updates-november-15-2024.md [hunk 9]
```diff
@@ -0,0 +1,94 @@
...
```
Comment (Hiviexd) [inline news/2024/2024-11-15-osulazer-updates-november-15-2024.md:80]: "btw" feels a bit too casual for the writing style of this newspost series

File: news/2024/2024-11-15-osulazer-updates-november-15-2024.md [hunk 10]
```diff
@@ -0,0 +1,94 @@
...
+
+![Copying a score link](/wiki/shared/news/2024-11-16-osulazer-updates-november-15-2024/score-link-copy.jpg)
+
+- **Option sliders can now be selected and adjusted with arrow keys.** They were previously controllable with arrow keys while hovered, but now they work exactly the same as in osu!(stable).
+
+![Option slider keyboard input](/wiki/shared/news/2024-11-16-osulazer-updates-november-15-2024/option-slider-keyboard-input.gif)
+
+---
+
+It's now easier than ever to farm performance points in [osu!(lazer)](https://osu.ppy.sh/home/download). If you were interested in the minor changes, check out the [full changelog](https://osu.ppy.sh/home/changelog/lazer/2024.1115.3).
```
Comment (Hiviexd) [inline news/2024/2024-11-15-osulazer-updates-november-15-2024.md:90]:
```suggestion
It's now easier than ever to farm performance points in [osu!(lazer)](https://osu.ppy.sh/home/download). If you are interested in every nook and cranny involving this update, check out the [full changelog](https://osu.ppy.sh/home/changelog/lazer/2024.1115.3).
```

File: news/2024/2024-11-15-osulazer-updates-november-15-2024.md (hunk 1 above)
Comment (Walavouchey) [reply news/2024/2024-11-15-osulazer-updates-november-15-2024.md:23]: it's actually supposed to be a noun phrase (and not a sentence), i.e. "added" is an adjective (but it's ambiguous and works both ways)

File: news/2024/2024-11-15-osulazer-updates-november-15-2024.md (hunk 2 above)
Comment (Walavouchey) [reply news/2024/2024-11-15-osulazer-updates-november-15-2024.md:24]: same here, these are worded as noun phrases

File: news/2024/2024-11-15-osulazer-updates-november-15-2024.md (hunk 9 above)
Comment (Walavouchey) [reply news/2024/2024-11-15-osulazer-updates-november-15-2024.md:80]:
```suggestion
- **New context menu option for copying link to an online score.** The same thing exists when right-clicking beatmaps in song select, by the way.
```

File: news/2024/2024-11-15-osulazer-updates-november-15-2024.md (hunk 7 above)
Comment (Walavouchey) [reply news/2024/2024-11-15-osulazer-updates-november-15-2024.md]: added

**Review by Hiviexd — APPROVED** [review]

**Pull request merged**